common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseCommand.java

```diff
@@ -24,20 +24,21 @@
 
 import androidx.annotation.NonNull;
 
+import com.microsoft.identity.common.exception.BaseException;
 import com.microsoft.identity.common.internal.request.OperationParameters;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public abstract class BaseCommand<T> implements Command<T> {
+public abstract class BaseCommand<T extends Object> implements Command<T> {
```

Comment:
I don't understand this bit -- my understanding is that `BaseCommand<T>` and `BaseCommand<T extends Object>` are synonymous -- what else could T possibly extend?